strawberryfields/gbs/data.py

```diff
@@ -440,7 +440,7 @@ class MoleculeDataset(Dataset, ABC):
         w (array): normal mode frequencies of the electronic ground state (:math:`\mbox{cm}^{-1}`)
         wp (array): normal mode frequencies of the electronic excited state (:math:`\mbox{cm}^{-1}`)
         Ud (array): Duschinsky matrix
-        d (array): Duschinsky displacement vector corrected with wp
+        delta (array): Duschinsky displacement vector

```

Comment:
Why do we make this change? I see `d` in `gbs_params` (https://github.com/XanaduAI/strawberryfields/blob/master/strawberryfields/gbs/vibronic.py)

Reply:
Ahh I guess because of https://github.com/XanaduAI/strawberryfields/pull/247 ?

Reply:
There are little inconsistencies in the definitions of Duschinsky displacement parameter. One way to deal with that is to use delta instead of d, or we have to explain the difference between delta and d all the time.